src/pages/creations/LogosCarousel/index.tsx

```diff
@@ -0,0 +1,301 @@
+import { styled } from 'styled-components';
+import { MOBILE_MAX_WIDTH, TABLET_MAX_WIDTH } from '~/components';
+import { Carousel } from 'react-responsive-carousel';
+import { useCallback, useEffect, useMemo, useState } from 'react';
```

Comment:
react imports should be on line 1

Reply:
https://github.com/defi-wonderland/web/pull/114/commits/bdf96f491f213c63ab3ba85c7e8f07cfdff5f739